recipes/biobox/meta.yaml

```diff
@@ -1,56 +0,0 @@
-{% set name = "biobox" %}
```

Comment:
Please don't remove other recipes in the folder. Only your recipe should be changed.

Reply:
There isn't another recipe, I made this recipe using one of my other recipes as a template

Reply:
But there is still something that is being deleted. Only your recipe should be in the PR, nothing else.